snippets/language-ruby.cson

```diff
@@ -114,6 +114,9 @@
   'def … end':
     'prefix': 'def'
     'body': 'def ${1:method_name}\n\t$0\nend'
+  'initialize':
+    'prefix': 'init'
+    'body': 'def initialize${1:(${2:arguments, …})}\n\t$0\nend'
```

Comment:
> No strong feelings about this though. I like my version (nested placeholders + more verbose suggestions) quite a lot, but still I am okay with whatever.

I like the nested placeholders as well (my comment wasn't about that, sorry if I was unclear). :+1: I was just thinking about the consistency of listing a single argument vs multiple arguments. I merged this but also tweaked the snippet so that it uses `${2:argument}` instead of `${2:arguments, …}`. Thanks for the contribution! :sparkle: